packages/desktop-electron/package.json

```diff
@@ -88,9 +85,9 @@
     "npmRebuild": false
   },
   "dependencies": {
+    "better-sqlite3": "^9.6.0",
```

Comment:
better-sqlite3 is an "external" dependency . It's specific to electron so should be defined here.